### PR TITLE
Adding arm64 architecture so it can be used on the M1 CircleCI hardware.

### PR DESCRIPTION
**Pull request by szweier**

### CODE REVIEW

File: .releaserc.json
Comment (kalkwarf) [inline .releaserc.json:7]: I'm not sure if we want to build fat, or just arm? My personal preference would be fat, but the trend on the internet appears to be platform specific, so as long as we can still build on Intel and Apple silicon at the same time, I guess this is fine.

**Review by kalkwarf — APPROVED** [review]

File: .releaserc.json
Comment (grinish21) [reply .releaserc.json:7]: go build natively does not support fat binaries. We would need to use something which is not part of go-build -> https://github.com/golang/go/issues/53281

**Review by grinish21 — APPROVED** [review]

**Comment by szweier** [comment]
@grinish21 would you be able to merge this when you have a moment, it looks like I don't have write access to be able to merge the PR even with approvals.

**Pull request merged**
